src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -172,9 +170,9 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
                 ? this.kernelProvider.get(activeEditor.document)
                 : undefined;
         if (kernel) {
-            const canStart = kernel.status !== ServerStatus.NotStarted;
+            const canStart = kernel.status !== 'unknown';
```

Comment:
+1

Reply:
Never liked our custom status type

Reply:
Was having a tough time when logging the kernel status from Jupyter last week.
Beacause I had to remember that `not started` === `unknown` in jupyter.

To be fair, having this enum doesn't hurt either,
The draw back is if and when jupyter changes their code, we have to change ours.
having wrappers shields us from that